test/parallel/test-v8-version-tag.js

```diff
@@ -7,9 +7,8 @@ const versionTag1 = v8.cachedDataVersionTag();
 assert.strictEqual(typeof versionTag1, 'number');
 assert.strictEqual(v8.cachedDataVersionTag(), versionTag1);
 
-// The value of cachedDataVersionTag is derived from the command line flags and
-// detected CPU features. Test that the value does indeed update when flags
-// are toggled.
+// The value is derived from the V8 version, command-line flags, and detected
```

Comment:
Nit: I think it makes sense to clarify that this comment refers to the value returned by `v8.cachedDataVersionTag()`. As is, a reader might wonder "what value?" as the comment is above `v8.setFlagsFromString('--allow_natives_syntax')` and not `v8.cachedDataVersionTag()`

Reply:
Sure, done. PTAL.